### DISCUSSION TITLE
事先準備的重要性

### DISCUSSION THREAD

**Discussion by tower20021202** [1101-GS4719-C/Python-Community-GS4719-C#1389] [General]
小時候上學時，家人常會說前一天睡覺前就要把隔天上學的書包準備好，我常覺得非常麻煩，效率不佳，但直到現在上學時，才發現事前準備的重要性，不然就有可能會像我一樣上課忘記帶筆電，又或是去辦護照忘記先準被好照片，這些都造成更大量時間浪費。不知大家有沒有其他例子來提醒我們事前準被的重要性？

**Comment by poweichen00** [comment]
我都會事先準備好隔天的東西，我之前有過當天一早要去比賽，想說再一起帶比賽的東西，結果少帶選手證，還要再跑一趟，造成更多麻煩。

**Comment by ALAN30359704** [comment]
我覺得事先準備如同課前預習一樣，可以更快進入狀況，熟悉未來會遇到問題。
